src/Web/Twitter/Feed.hs

```diff
@@ -43,13 +44,21 @@ timeline oauth credential count excludeReplies username = do
       Right ts     -> Right $ map (simplifyTweet . linkifyTweet) ts
 
 createRequest :: String -> Int -> Bool -> IO Request
-createRequest username count excludeReplies = parseUrl $ timelineUrl username count excludeReplies
+createRequest username count excludeReplies = parseUrlThrow $ timelineUrl username count excludeReplies
+#if (!MIN_VERSION_http_conduit(2,1,11))
+  where
+    parseUrlThrow = parseUrl
+#endif
 
 getResponse :: OAuth -> Credential -> Request -> IO (Response BS.ByteString)
-getResponse oauth credential req =
-  withManager $ \m -> do
-    signedreq <- signOAuth oauth credential req
-    httpLbs signedreq m
+getResponse oauth credential req = do
+  m <- newManager tlsManagerSettings
+  signedreq <- signOAuth oauth credential req
+  httpLbs signedreq m
+#if (!MIN_VERSION_http_conduit(2,1,11))
+    where
+      tlsManagerSettings = defaultManagerSettings
```

Comment:
I'm not sure if adding more dependencies to the project is better. What could be the benefits about that? And, what could be the possible problems using CPP options?

I'll take a look at `defaultManagerSettings` and `tlsManagerSettings` differences.